transmart-rest-api/grails-app/controllers/org/transmartproject/interceptors/ApiAuditInterceptor.groovy

```diff
@@ -30,24 +36,20 @@ class ApiAuditInterceptor {
         match(controller: ~/version/)
     }
 
-    boolean before() { true }
-
     boolean after() {
         report("$controllerName request", eventMessage)
     }
 
     protected boolean report(String event, String eventMessage) {
-
-        accessLogService.report(
-                authContext?.user,
+        accessLogEntryResource.report(
+                authContext.user,
```

Comment:
there was an issue with null user in the log entry. Do you know what was the reason? It was not related to the authContext being null?

Reply:
Yes, adding `@Autowired` to the authContext seemed to solve that issue. Apparently the bean was not automatically injected.